docs/core/tools/dotnet-test.md

```diff
@@ -380,8 +400,183 @@ You can enclose expressions in parenthesis when using conditional operators (for
 
 For more information and examples on how to use selective unit test filtering, see [Running selective unit tests](../testing/selective-unit-tests.md).
 
-## See also
+#### See also
 
 - [Frameworks and Targets](../../standard/frameworks.md)
 - [.NET Runtime Identifier (RID) catalog](../rid-catalog.md)
 - [Passing runsettings arguments through commandline](https://github.com/microsoft/vstest/blob/main/docs/RunSettingsArguments.md)
+
+### [dotnet test with MTP](#tab/dotnet-test-with-mtp)
+
+**This article applies to:** ✔️ .NET 10 SDK and later versions
+
+#### Synopsis
+
+```dotnetcli
+dotnet test
+    [-a|--arch <ARCHITECTURE>]
+    [-c|--configuration <CONFIGURATION>]
+    [--directory <DIRECTORY_PATH>]
+    [-mptm|--max-parallel-test-modules <NUMBER>]
```

Comment:
I would avoid short names for our options, this has only been causing issues in past.